src/main.cc

```diff
@@ -47,7 +47,7 @@ int wmain(int argc, const wchar_t* argv[]) {
 
       unsigned short v1, v2, v3, v4;
       if (!parse_version_string(argv[++i], &v1, &v2, &v3, &v4))
-        return print_error("Unable to parse version string");
+        return print_error("Unable to parse version string for FileVersion");
```

Comment:
> Why is adding one more line a concern?

Mostly because it messes with the code style, and I wanted this PR to be minimally invasive.

> I'm confused. Why do you need a function taking a variable number of arguments?

`print_error` is used elsewhere with only a single argument. This would be a change to that to have a second, optional argument (and I don't even remember how that works in C++). Again, I'd like my changes to be minimally invasive.

I'd be happy to look at a PR on my PR to update it to your liking. I just don't have time to research and write the code myself at the moment.